psydac/linalg/utilities.py

```diff
@@ -13,46 +14,64 @@
 )
 
 #==============================================================================
-def array_to_psydac(x, Xh):
-    """ converts a numpy array to StencilVector or BlockVector format"""
+def array_to_psydac(x, V):
+    """ 
+    Convert a NumPy array to a Vector of the space V. 
+    This function works in parallel but it is very costly and should be avoided if performance is a priority.
```

Comment:
We should mention that this function is designed to be the inverse of the `toarray()` method of our `Vector` objects.

Reply:
Done in [1bf8f22](https://github.com/pyccel/psydac/pull/404/commits/1bf8f221b454c92dacfafa505bcf03c34ef03827)